packages/vaadin-button/theme/lumo/vaadin-button-styles.js

```diff
@@ -233,13 +233,15 @@ registerStyles(
 
     /* Icons */
 
+    [part] ::slotted(vaadin-icon),
     [part] ::slotted(iron-icon) {
```

Comment:
Do we still support `iron-icon`? Is it needed for the Flow counterpart?

Reply:
Yes, `IronIcon` is deprecated but not removed https://github.com/vaadin/flow-components/issues/884